plugin/pkg/admission/priority/admission.go

```diff
@@ -252,12 +240,5 @@ func (p *PriorityPlugin) getDefaultPriority() (int32, error) {
 	if dpc != nil {
 		priority = dpc.Value
```

Comment:
here we should just:
```
return dpc.value, nil
```
and at the end of the function
```
return int32(scheduling.DefaultPriorityWhenNoDefaultClassExists), nil
```

Reply:
Done